src/discrete.jl

```diff
@@ -1,8 +1,95 @@
+"""
+The structue to represent a discrete dynamical system
+with respect to *shift*. Internally just stores an ODE structur
```

Comment:
Internally just stores an `ODE` ?

Reply:
Yes, this seems to be perfectly sufficient for now. Since the structure is used via functions (`x_vars`, etc), the internal details can be changed later if necessary